Prototypes/freestyle-wg-probe/bridge.go

```
@@ -0,0 +1,189 @@
+	if err != nil {
+		return bridgeInfo{}, err
+	}
+	pid, err := strconv.Atoi(strings.TrimSpace(out))
+	if err != nil {
+		return bridgeInfo{}, fmt.Errorf("remote bridge did not return a process id: %w", err)
+	}
+	deadline := time.Now().Add(30 * time.Second)
+	for time.Now().Before(deadline) {
+		select {
+		case <-ctx.Done():
+			_, _ = cloud.exec(context.Background(), vmID, fmt.Sprintf("kill -TERM %d 2>/dev/null || true; rm -f %s %s", pid, shellQuote(marker), shellQuote(tokenPath)))
```

golangci-lint (2.13.2)

[error] Non-inherited new context, use function like `context.WithXXX` instead

(contextcheck)